pkg/mobile/secrets.go

```diff
@@ -18,3 +20,15 @@ func NewSecretsCRUDL(client kubev1.CoreV1Interface) *SecretsCRUDLImpl {
 func (crudl *SecretsCRUDLImpl) Create(namespace string, secret *v1.Secret) (*v1.Secret, error) {
 	return crudl.secretsClient.Secrets(namespace).Create(secret)
 }
+
+func (crudl *SecretsCRUDLImpl) List(namespace string) (*v1.SecretList, error) {
+	listOpts := metav1.ListOptions{}
+	return crudl.secretsClient.Secrets(namespace).List(listOpts)
+}
+
+func (crudl *SecretsCRUDLImpl) Watch(namespace string) func() (watch.Interface, error) {
+	return func() (watch.Interface, error) {
+		watchOpts := metav1.ListOptions{}
```

Comment:
we can probably use the `LabelSelector` in the `ListOptions` to only watch the secrets that we are interested in.

Reply:
ok, will add it